spec/lib/bundle_context_temporary_spec.rb

```diff
@@ -1,6 +1,8 @@
 RSpec.describe BundleContextTemporary do
   let(:revs_context) { context_from_proj(:proj_revs) }
 
+  before { allow_any_instance_of(described_class).to receive(:validate_usage) } # to be replaced w/ AR validation
```

Comment:
was this allowance added because previously valid fixture configs are no longer valid?

Reply:
right.  still using the *rest* of the fixture, but strictly speaking it isn't valid anymore.